app/src/main/java/com/kyuwon/booklog/service/posts/PostsService.java

```diff
@@ -19,10 +21,28 @@
 
     /**
      * 게시물 정보를 받아 저장하고 등록된 게시물을 리턴한다.
+     *
      * @param postsSaveRequestData 게시물 정보
      * @return 등록된 게시물
      */
     public Posts save(PostsSaveRequestData postsSaveRequestData) {
         return postsRepository.save(postsSaveRequestData.toEntity());
     }
+
+    /**
+     * 게시물 수정 정보를 받아 수정하고 수정된 게시물을 리턴한다.
+     *
+     * @param id          해당 게시물 id
+     * @param requestData 수정할 정보
+     * @return 수정된 게시물
+     */
+    public Posts update(Long id,
+                        PostsUpdateRequestData requestData) {
```

Comment:
여기에서는 인터페이스를 받으세요. 이렇게 하면 인터페이스에 선언된 값 제공 함수만 서비스에서 사용할 수 있게 됩니다.

```suggestion
                        PostUpdateRequest requestData) {
```

Reply:
수정했습니다! 누락된 부분들이 있었네요.